resources/scripts/jquery/jquery-utils.js

```diff
@@ -308,6 +308,30 @@
     	$(".eXist_wizard_next", container).click(function () {
     		if (currentPage == pages.length - 1)
     			return;
+		var name = document.getElementsByName('name')[0].value;
+           	var abbrev = document.getElementsByName('abbrev')[0].value;
+           	var title = document.getElementsByName('title')[0].value;
+
+                if (name.startsWith('urn:') || name.startsWith('file:')) {
```

Comment:
URNs _are_ allowed package names. An example URN: `urn:uuid:6e8bc430-9c3a-11d9-9669-0800200c9a66`

Have you considered an alternative to `alert` to present these errors to the user?
In eXide errors are shown as a toast message in the bottom right corner. But that is me assuming this code is (only) used in that context.

Reply:
OK, I was confused by your 'the URN syntax would not be supported'. I've taken that check out. I think this code is only used here. I think in this specific case the alert is a better option.